app/categorization/work.py

```diff
@@ -0,0 +1,88 @@
+from datetime import datetime
+import json
+from typing import Dict, Any, Union
+from langchain_community.chat_models import ChatOllama
+from langchain_core.prompts import PromptTemplate
+from langchain.chains import SequentialChain
+
+from llm_helper import (
+    is_years,
+    is_bounded,
+    is_european_decimal,
+    is_float,
+    is_integer,
+    is_iso8601,
+   
+    SexLevel,
+    AgeFormat
+
+)
+from promptTemplate import(
+    prompt,
+    Aprompt,
+    Dprompt
```

Comment:
```suggestion

```
These are also not needed since they are invoked in the llm_incovation2 that is called from that script right?